test/functional/testlib.bash

```diff
@@ -961,17 +961,51 @@ bump_format() {
 	new_version="$((version+20))"
 	new_version_path="$env_name"/web-dir/"$new_version"
 	format="$(< "$env_name"/web-dir/"$version"/format)"
-	mom="$new_version_path"/Manifest.MoM
 
 	# create two new versions for the format bump, each version separated by 10
 	# from each other and from the current latest version
 	create_version "$env_name" "$middle_version" "$version" "$format"
 	create_version -r "$env_name" "$new_version" "$middle_version" "$((format+1))"
+
+	# Update the +10 version
+	mom="$middle_version_path"/Manifest.MoM
+
+	# +10 and +20 versions should have matching content
+	sudo rm -rf "$middle_version_path"
+	sudo cp -r "$new_version_path" "$middle_version_path"
+
+	# Update +10 format and versions
+	update_manifest -p "$mom" format "$((format))"
```

Comment:
I guess you could have just used "$format" here, since this is just a variable not an expression I don't think you need the (( ))